test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -1958,6 +1958,161 @@ public async Task DependencyGraphRestoreUtility_LegacyPackageRef_Restore_BuildTr
+                    projectServices.SetupInstalledPackages(
+                        NuGetFramework.Parse(projectTargetFrameworkStr),
+                        new LibraryDependency
+                        {
+                            LibraryRange = new LibraryRange(
+                                packageA.PackageId,
+                                versionRange: null,
+                                LibraryDependencyTarget.Package),
+                        });
+
+                    var legacyPRProject = new LegacyPackageReferenceProject(
+                        vsProjectAdapter,
+                        Guid.NewGuid().ToString(),
+                        projectServices,
+                        _threadingService);
+                    testSolutionManager.NuGetProjects.Add(legacyPRProject);
+
+                    var packageContextB = new SimpleTestPackageContext(packageB.PackageId, packageB.Version);
+                    packageContextB.Files.Clear();
+                    packageContextB.AddFile("build/packageB.targets");
+                    packageContextB.AddFile("build/packageB.props");
+                    packageContextB.AddFile("buildCrossTargeting/packageB.targets");
+                    packageContextB.AddFile("buildCrossTargeting/packageB.props");
+                    packageContextB.AddFile("buildTransitive/packageB.targets");
+                    packageContextB.AddFile("lib/net45/b.dll");
+                    await SimpleTestPackageUtility.CreateFullPackageAsync(packageSource, packageContextB);
+
+                    var packageContextA = new SimpleTestPackageContext(packageA.PackageId, packageA.Version);
+                    packageContextA.Files.Clear();
+                    packageContextA.AddFile("build/packageA.targets");
+                    packageContextA.AddFile("build/packageA.props");
+                    packageContextA.AddFile("buildCrossTargeting/packageA.targets");
+                    packageContextA.AddFile("buildCrossTargeting/packageA.props");
+                    packageContextA.AddFile("buildTransitive/packageA.targets");
+                    packageContextA.AddFile("lib/net45/a.dll");
+                    packageContextA.Dependencies.Add(packageContextB);
+
+                    await SimpleTestPackageUtility.CreateFullPackageAsync(packageSource, packageContextA);
+
+                    var restoreContext = new DependencyGraphCacheContext(testLogger, testSettings);
+                    var providersCache = new RestoreCommandProvidersCache();
+
+                    var dgSpec = await DependencyGraphRestoreUtility.GetSolutionRestoreSpec(testSolutionManager, restoreContext);
+
+                    // Act
+                    var restoreSummaries = await DependencyGraphRestoreUtility.RestoreAsync(
+                        testSolutionManager,
+                        dgSpec,
+                        restoreContext,
+                        providersCache,
+                        (c) => { },
```

Comment:
Can you name this argument, please. Without looking at the definition of `RestoreAsync`, I have no idea why a empty anonymous function is being passed in.  Also consider using a discard `_` instead of an unused named variable `c`.

Reply:
Updated.